app/build.gradle

```diff
@@ -170,7 +170,7 @@ android {
         niger.assets.srcDir 'src/niger/assets'
 
         tchadDev.java.srcDir 'src/tchad/java'
-        tchadDev.res.srcDir 'src/tchad/res'
+        tchadDev.res.srcDirs = ['src/tchad/res', 'src/localeBepha/res']
```

Comment:
@malinowskikam Is 'src/localeBepha/res' the right folder? I see the Tchad res in 'src/localeTchad/res'?